04-vpcs/topic4.txt

```diff
@@ -0,0 +1,158 @@
+Lab 4.1.4: Test Instance
+
+Launch an EC2 instance into your VPC.
+
+    Create another CFN template that specifies an EC2 instance.
+
+    For the subnet and VPC, reference the outputs from your VPC stack.
+
+    Use the latest Amazon Linux AMI.
+
+    Create a new parameter file for this template. Include the EC2 AMI ID, a T2 instance type, and the name of your key pair.
+
+    Provide the instance ID and private IP address as stack outputs.
+
+    Use the same tags you put on your VPC.
+
+After you launch your new stack, can you ssh to the instance?
+
+===
+Nope, no connectivity is provisioned as of yet
+===
+
+Is there a way that you can verify Internet connectivity from the instance without ssh'ing to it?
+
+===
+Ping it.
```

Comment:
So this is not necessarily correct, you are verifying your connectivity to the instance with a ping to the instance, in order to verify the instance has internet you would need to ping from the instance.